src/components/map/layers/NodeLayer.tsx

```diff
@@ -57,6 +57,10 @@ export default class NodeLayer extends Component<MarkerLayerProps> {
             html = this.getMarkerHtml(isSelected ? s.disabledMarkerHighlight : s.disabledMarker);
         }
 
+        if (this.props.nodeStore!.isNodeTimeAlignmentStop(node.id)) {
+            html = this.getMarkerHtml(s.timeAlignmentMarker);
+        }
+
```

Comment:
Alkuun:
let type;
        if (this.props.nodeStore!.isNodeDisabled(node.id)) {
            type = NodeType.DISABLED;
        } else if (this.props.nodeStore!.isNodeTimeAlignmentStop(node.id)) {
            type = NodeType.TIME_ALIGNMENT;
        } else {
            type = node.type;
        }